### ISSUE TITLE
Weekly Template tags are off

### ISSUE THREAD

**Issue by etiennepellegrini** [liamcain/obsidian-calendar-plugin#78]
I use the template tags in order to insert backlinks to each daily note for the weekly note, as such:
```
Backlinks: [[{{monday:YYYY-MM-DD_ddd}}|Monday]], [[{{tuesday:YYYY-MM-DD_ddd}}|Tuesday]], [[{{wednesday:YYYY-MM-DD_ddd}}|Wednesday]], [[{{thursday:YYYY-MM-DD_ddd}}|Thursday]], [[{{friday:YYYY-MM-DD_ddd}}|Friday]], [[{{saturday:YYYY-MM-DD_ddd}}|Saturday]], [[{{sunday:YYYY-MM-DD_ddd}}|Sunday]]
```

However, I get the following results when creating a new weekly note:
- with week starting on Sunday:
```
Backlinks: [[2020-12-20_Sun|Monday]], [[2020-12-21_Mon|Tuesday]], [[2020-12-22_Tue|Wednesday]], [[2020-12-23_Wed|Thursday]], [[2020-12-24_Thu|Friday]], [[2020-12-25_Fri|Saturday]], [[2020-12-26_Sat|Sunday]]
```
- with week starting on Monday:
```
Backlinks: [[2020-12-22_Tue|Monday]], [[2020-12-23_Wed|Tuesday]], [[2020-12-24_Thu|Wednesday]], [[2020-12-25_Fri|Thursday]], [[2020-12-26_Sat|Friday]], [[2020-12-27_Sun|Saturday]], [[2020-12-21_Mon|Sunday]]
```

For now, my workaround is to start the backlinks at Sunday even tho it's going to become a link to Monday:
with week starting on Monday:
`[[{{sunday:YYYY-MM-DD_ddd}}|Monday]]` --> ` [[2020-12-21_Mon|Monday]]`

**Comment by liamcain** [comment]
Great catch, sorry about that! Will have a fix out shortly.

**Comment by liamcain** [comment]
Hey, @etiennepellegrini I just released v1.4.7 which should fix this issue (sorry for the delay, was hoping to include it as part of a bigger release but decided to just get some bug fixes in first)

**Comment by etiennepellegrini** [comment]
Works like a charm thanks a lot!
